File: programme/alphabet-triangle-pattern/AlphabetTrianglePattern.cs
// Write a program to print alphabet triangle pattern

using System;

public class AlphabetTrianglePattern
{
    private static void pattern(int n)
    {
        Console.WriteLine("Output :");
        for (int i = 1; i <= n; i++)
        {
            int c = (int)('A');
            for (int k = n; k > i; k--)
            {
                Console.Write(" ");
            }
            for (int j = 1; j < 2 * i; j++)
            {
                Console.Write(((char)c));
                if (j < i)
                {
                    c++;
                }
                else
                {
                    c--;
                }
            }
            Console.WriteLine();
        }
    }
    public static void Main(String[] args)
    {
        Console.Write("Input  : ");
        int n = int.Parse(Console.ReadLine());
        pattern(n);
    }
}

File: programme/alphabet-triangle-pattern/alphabet-triangle-pattern.jl
# WAP to get the alphabet triangle pattern


function makePattern(num)
    print("Output : ")

    for i in 1:num
       c= 65
       
       # Creates the spaces before the letters
       println("") 
        for k in num-1:-1:i
        print(" ") 
    end
        
        # Fills up letters 
        for j in 1:(2*i - 1)
            
            if j < i
                print(Char(c))
                c += 1
            else
                print(Char(c))
                c -= 1
            end
        end        
    end        
end

print("Input : ")
len = parse(Int32, readline())


makePattern(len)